pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -40,6 +40,9 @@ import (
 	"github.com/tigera/operator/pkg/controller/utils"
 	"github.com/tigera/operator/pkg/controller/utils/imageset"
 	"github.com/tigera/operator/pkg/render"
+	relasticsearch "github.com/tigera/operator/pkg/render/common/elasticsearch"
```

Comment:
same here - why do we need to alias everything with an "r"?

Reply:
The "r" is "render", I thought it would be nicer to have that the name consistent and differentiate between the elasticsearch package which actually does stuff with elasticsarch. It's kind of like how we alias the k8s core package from /v1 to corev1 even if there's nothing conflicting.